File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-8.js
const prompt = require('prompt-sync')();

const a = prompt('Digite o valor de A: ')
const b = prompt('Digite o valor de B: ')
const c = prompt('Digite o valor de C: ')

const descriminante = (b**2) - (4*a*c)

if(descriminante > 0){
    console.log(`\nDescriminante:${descriminante}`)
    console.log(`X:Positivo = ${(-b + Math.sqrt(descriminante))/(2*a)}`)
    console.log(`X:Negativo = ${(-b - Math.sqrt(descriminante))/(2*a)}`)
}else{
    console.log(`\nDescriminante:${descriminante}`)
    console.log('Não existem raízes reais')
}

File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_8.py
from math import sqrt

a = int(input('Digite o valor de a:'))
b = int(input('Digite o valor de b:'))
c = int(input('Digite o valor de c:'))

descriminante = (b**2) - (4*a*c)


if descriminante < 0:
    print('Não existem raízes reais.')
else:
    x_positivo = (-b + sqrt(descriminante)) / (2*a)
    x_negativo = (-b - sqrt(descriminante)) / (2*a)

    print(f'*** Raízes reais da equação ***\n'
          f'X positivo:{x_positivo}\n'
          f'X negativo:{x_negativo}')
    
    print(f'Descriminante:{descriminante}') # Coloquei só para ver o valor, não era necessário.

